Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/notification/DigitalCovidCertificateNotifications.kt

```diff
@@ -50,7 +51,7 @@ class DigitalCovidCertificateNotifications @Inject constructor(
             setSmallIcon(R.drawable.ic_notification_icon_default_small)
             priority = NotificationCompat.PRIORITY_DEFAULT
 
-            val pendingIntent = NavDeepLinkBuilder(context)
+            val pendingIntent = navDeepLinkBuilderFactory.create(context)
                 .setGraph(R.navigation.trace_location_attendee_nav_graph)
                 .setComponentName(MainActivity::class.java)
                 .setDestination(R.id.checkInsFragment)
```

Comment:
Just noticed this here, although you did not change this here. Seems wrong to me that the DCC notification goes to the checkInsFragment??

Reply:
The destination is overwritten in code, but the graph is wrong, strange...